biz.aQute.bndlib/src/aQute/bnd/osgi/AnnotationHeaders.java

```diff
@@ -115,6 +117,11 @@
 	static final String				STD_HEADER				= "org.osgi.annotation.bundle.Header";
 	static final String				STD_HEADERS				= "org.osgi.annotation.bundle.Headers";
 
+	// Used to detect attributes and directives on Require-Capability and
+	// Provide-Capability
+	static final String				ATTRIBUTE			= "org.osgi.annotation.bundle.Attribute";
```

Comment:
Are these not to be prefixed with `STD_` like the other bundle annotation type names?

Reply:
Will do - the previous ones did to differentiate them from the bnd equivalents. There are no bnd equivalents for `@Attribute` or `@Directive` but it probably still helps for the constant names to be internally consistent.